
This works around breakage of "terminal_output gfxterm".  IT SHOULD NOT
GO UPSTREAM.

I'll try to figure out what happened to the terminal_output command..

  -- rmh

Index: util/grub.d/00_header.in
===================================================================
--- util/grub.d/00_header.in	(revision 2443)
+++ util/grub.d/00_header.in	(working copy)
@@ -91,7 +91,7 @@
   set gfxmode=${GRUB_GFXMODE}
   insmod gfxterm
   insmod ${video_backend}
-  if terminal_output gfxterm ; then true ; else
+  if terminal_output.gfxterm ; then true ; else
     # For backward compatibility with versions of terminal.mod that don't
     # understand terminal_output
     terminal gfxterm
@@ -104,7 +104,7 @@
   ;;
   x*)
     cat << EOF
-if terminal_output ${GRUB_TERMINAL_OUTPUT} ; then true ; else
+if terminal_output.${GRUB_TERMINAL_OUTPUT} ; then true ; else
   # For backward compatibility with versions of terminal.mod that don't
   # understand terminal_output
   terminal ${GRUB_TERMINAL_OUTPUT}
